src/deploy/schema.json

```diff
@@ -3,6 +3,10 @@
   "title": "schema",
```

Comment:
Use [conventional commits](https://www.conventionalcommits.org/en/v1.0.0-beta.2/) on your commit message
`feat: add a packageVersion to options`